src/Symfony/Component/Config/Builder/ConfigBuilderGenerator.php

```diff
@@ -89,7 +89,7 @@ private function getFullPath(ClassBuilder $class): string
     private function writeClasses(NodeInterface $node): void
```

Comment:
the unused `$node` argument should be removed here

Reply:
Indeed, thanks